src/resizable-card-layout.scss

```diff
@@ -112,6 +118,7 @@ $block: #{$fd-namespace}-resizable-card-layout;
 
     &--horizontal {
       // subtract resize-corner icon font-size/2
+      z-index: 2;
```

Comment:
z-index is used 3 times, it should be moved to a variable

Reply:
used variable now

Reply:
used variable now